README.md

```diff
@@ -162,7 +163,13 @@ Then consumers of the library could just express the need for a maven pod with d
             """
         }
     }
+
+#### Using a different namespace
  
+There might be cases, where you need to have the slave pod run inside a different namespace than the one jenkins is running in.
```

Comment:
IIRC pods are started in the namespace in the cloud definition, not in the one jenkins is running